### PR TITLE
Deploy new amm

### PR DESCRIPTION
**Pull request by tensojka**
- [x] deploy AMM
- [x] deploy lptokens
- [x] deploy options 
- [x] add tests
- [] also test that you can trade bitcoin

### CODE REVIEW

**Comment by tensojka** [comment]
So far this is based on snforge-tests branch, before merge and deployment after this is tested I will rebase on top of master

File: src/deploy_amm.cairo [hunk 1]
```diff
@@ -0,0 +1,91 @@
+mod Deploy_AMM {
```
Comment (DaveVodrazka) [inline src/deploy_amm.cairo:1]: Use PascalCase `DeployAMM`

File: src/deploy_amm.cairo [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        symbol: felt252,
+        quote_token_address: ContractAddress,
+        base_token_address: ContractAddress,
+        option_type: OptionType,
+        voladjspd: Fixed
+    ) -> ContractAddress {
+        let lptoken_class: ClassHash = LP_TOKEN_CLASS_HASH.try_into().unwrap();
+        let mut lpt_calldata: Array<felt252> = ArrayTrait::<felt252>::new();
+        lpt_calldata.append('Carmine ETH/USDC call pool');
+        lpt_calldata.append('C-ETHUSDC-C');
```
Comment (MarekHauzr) [inline src/deploy_amm.cairo:105]: shouldn't these two add name and symbol?

Comment (tensojka) [reply src/deploy_amm.cairo:105]: yes, thanks

**Comment by tensojka** [comment]
(it's cumbersome to run tests in CI as you have to compile snforge from source now)

deploy amm + deposit to liqpool works

**Pull request merged**